src/app/App.js

```diff
@@ -91,13 +141,22 @@ export default function App() {
     )
     function hasSelectedOption(tagList, selection) {
       const matchedOption = selection.filter(
-        option => tagList.indexOf(option) >= 0
+        option => tagList.indexOf(option) !== -1
       )
       return matchedOption.length > 0
     }
     return filteredPlants
   }
 
+  // übergebene tag entspricht der option.id
+  function getOptionLabel(tag) {
+    const optionGroupArray = getOptionGroupsByOptionId(tag)
+    const optionArray = optionGroupArray[0].options.filter(
```

Comment:
does it make sense to filter an array that contains only one object?. mabye you can pick the label immediately with
`optionGroupArray[0].options.label`